domains/ttos.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ttos030",
+        "email": "thomasfacebook90@gmail.com"
+    },
+    "record": {
+        "URL": "https://hk.yahoo.com/?p=us"
```

Comment:
this literally just redirects to Yahoo Hong Kong, can you like add your website in the record?